source/accessibility.html.md.erb

```diff
@@ -25,9 +25,7 @@ We’ve also made the website text as simple as possible to understand.
 
 ### How accessible this website is
 
-We know some parts of this website are not fully accessible for the following reasons:
-
-- there are issues caused by our Technical Documentation Template
+This website is fully compliant with the Web Content Accessibility Guidelines version 2.1 AA standard.
```

Comment:
add in link to https://www.w3.org/TR/WCAG21/ i.e. "This website is fully compliant with the [Web Content Accessibility Guidelines version 2.1](https://www.w3.org/TR/WCAG21/) AA standard."